doc/ug/dv_methodology.md

```diff
@@ -415,6 +415,36 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
+### X-Propagation (Xprop)
+
+Standard RTL simulations ignore the uncertainty of X-valued control signals and assign predictable output values.
+As a result, RTL simulations often fail to detect design problems related to the lack of X-Propagation, which actually can be detected in gate-level simulations.
+With Xprop in RTL simulations, we can detect these problems without running gate-level simulation.
+
+It's mandatory to enable Xprop when running regression for coverage closure.
+VCS provides 3 modes for Xprop.
+  * **xmerge**: More pessimistic than a standard gate-level simulation
```

Comment:
I think it will be good to add the mux SV code example here and the truth table for all these modes - it is extremely helpful in understanding what we are talking about here. 
https://www.verilogpro.com/x-propagation-with-vcs-xprop/

Reply:
Added and changed to use pessimistic mode